docker-compose.yml

```diff
@@ -11,15 +9,15 @@ services:
     environment:
       SSH_KEY: ${CKAN_NETKAN_SSHKEY}
       CKANMETA_REMOTES: ${CKAN_METADATA_PATHS}
-      CKANMETA_USER: ${CKAN_METADATA_USER}
-      CKANMETA_REPOS: ${CKAN_METADATA_REPOS}
+      CKAN_USER: ${CKAN_METADATA_USER}
+      CKAN_REPOS: ${CKAN_METADATA_REPOS}
```

Comment:
Why was this changed? It looks like it was correct before, there's no obvious spot where the new name is being referenced, and the old names are still live later in the file, so this change makes things less consistent.

https://github.com/KSP-CKAN/NetKAN-Infra/blob/c78d260c2f9adc6cf87d3dd91949798e5ef4a85a/docker-compose.yml#L91-L92

Reply:
Because they didn't work during my sanity check.

https://github.com/KSP-CKAN/NetKAN-Infra/blob/master/netkan/netkan/cli/common.py#L42

Reply:
OK, should we update the other occurrences then? Lines 91–92?